guides/src/content/release_notes/3_5_0.md

```diff
@@ -117,7 +117,7 @@ Also please review each of the noteworthy changes, and ensure your customization
 or extensions are not effected. If you are affected by a change, and have any
 of your own tips please submit a PR to help the next person!
 
-- Added `Address#EXCLUDED_KEYS_FOR_COMPARISION` so developers won't need to rewrite `Address#same_as` method
+- Added `Address#EXCLUDED_KEYS_FOR_COMPARISON` so developers won't need to rewrite `Address#same_as` method
```

Comment:
@MatthewKennedy let's not change it here, we would require this change to be listed in 4.2.0 notes so developers would know they will need to update their code :)

Reply:
My mistake, I'll get that sorted.